apps/fluent-tester/src/TestComponents/Tooltip/TooltipDefault.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { View } from 'react-native';
+
+import { Button } from '@fluentui-react-native/experimental-button';
+import { Tooltip } from '@fluentui-react-native/tooltip';
+
+export const TooltipDefault: React.FunctionComponent = () => {
+  return (
+    <View>
+      <Tooltip content="Test" positioning="topCenter">
+        <Button>Test</Button>
+      </Tooltip>
```

Comment:
suggestion: Positioning could be its own test section

Reply:
Updated!